groups/rds/profiles/heritage-staging-eu-west-2/vars

```diff
@@ -0,0 +1,115 @@
+# Account details
+aws_profile = "heritage-staging-eu-west-2"
+aws_region  = "eu-west-2"
+aws_account = "heritage-staging"
+
+# Account shorthand
+account     = "hstg"
+region      = "euw2"
+environment = "staging"
+
+# RDS Instance settings
+identifier                   = "smartview"
+name                         = "smartstg"
+instance_class               = "db.m5.large"
+allocated_storage            = 140
+backup_retention_period      = 2
+multi_az                     = true
+rds_maintenance_window       = "Wed:00:00-Wed:03:00"
+rds_backup_window            = "03:00-06:00"
+performance_insights_enabled = false
+
+# RDS Engine settings
+major_engine_version       = "12.1"
+engine_version             = "12.1.0.2.v24"
+license_model              = "license-included"
+auto_minor_version_upgrade = true
+
+rds_onpremise_access = [
+  "10.65.2.0/24",
+  "10.65.66.0/24",
+  "10.65.130.0/24",
+  "192.168.70.0/24",
+  "192.168.90.0/24"
+]
```

Comment:
I wouldn't include these here (public).
Can these values come from Vault or a remote state?

Reply:
PS - similar to what was done with `admin_cidrs`

Reply:
Updated in latest commits